books/RayTracingTheRestOfYourLife.html

```diff
@@ -2405,73 +2405,8 @@
 
 
 
-Acknowledgments
-====================================================================================================
-
-<div class="credit-list"> **Original Manuscript Help**
-
-  - Dave Hart
-  - Jean Buckley
-</div>
-
-<div class="credit-list"> **Web Release**
-
-  - Berna Kabadayı
-  - Lorenzo Mancini
-  - Lori Whippler Hollasch
-  - Ronald Wotzlaw
-</div>
-
-<div class="credit-list"> **Corrections and Improvements**
-
-  - Aaryaman Vasishta
-  - Andrew Kensler
-  - Apoorva Joshi
-  - Aras Pranckevičius
-  - Becker
-  - Ben Kerl
-  - Benjamin Summerton
-  - Bennett Hardwick
-  - Dan Drummond
-  - David Chambers
-  - David Hart
-  - Eric Haines
-  - Fabio Sancinetti
-  - Filipe Scur
-  - Frank He
-  - Gerrit Wessendorf
-  - Grue Debry
-  - Ingo Wald
-  - Jason Stone
-  - Jean Buckley
-  - Joey Cho
-  - Lorenzo Mancini
-  - Marcus Ottosson
-  - Matthew Heimlich
-  - Nakata Daisuke
-  - Paul Melis
-  - Phil Cristensen
-  - Ronald Wotzlaw
-  - Shaun P. Lee
-  - Tatsuya Ogawa
-  - Thiago Ize
-  - Vahan Sosoyan
-</div>
-
-<div class="credit-list"> **Tools**
-
-  <div class="indented">
-  Thanks to the team at [Limnu][] for help on the figures.
-
-  Huge shout out to Morgan McGuire for his fantastic [Markdeep][] library.
-  </div>
-</div>
-
-
+                               (insert acknowledgments.md.html here)
```

Comment:
Is this adequate?

Reply:
Yup. Classic Markdeep magical incantation.

Reply:
Similar to how “... the acknowledgments section ...” gets an automagic link to the section.